pnpm-workspace.yaml

```diff
@@ -2,3 +2,4 @@ packages:
   - 'packages/**'
   - 'examples/react/**'
   - 'examples/solid/**'
+  # - 'examples/vue/**'
```

Comment:
why is this commented out? I think we'd need it so that pnpm installs dependencies for all examples?

Reply:
Right now it does not work, cause `@tanstack/vue-query` does not exist and pnpm fails on install

Reply:
oh shoot yeah that makes sense. ok then let's please create a follow-up PR after we release vue-query that enables this :)

Reply:
I wonder though if we could do the same trick as with referencing core - `"@tanstack/query-core": "workspace:*",` in examples? 🤔

Reply:
I thought about this, but I don't think so because examples need to work in isolation in codesandbox...